### PR TITLE
fix(projectgridcard): Fix messages and date formatting

### PR DESCRIPTION
**Pull request by lukasbals**

### CODE REVIEW

File: src/components/ProjectGridCard/ProjectGridCard.tsx [hunk 1]
```diff
@@ -3,9 +3,9 @@ import React, { useContext } from 'react';
 import Image from 'next/image';
 import PortfolioProject from '../../models/PortfolioProject';
 import { strapiMediaUrl } from '../..';
-import { IntlContext } from 'react-intl';
```
Comment (lukasbals) [inline src/components/ProjectGridCard/ProjectGridCard.tsx:6]: This was the main problem with the messages. The `IntlContext` was imported from `react-intl` and not from our `ContextProvider`.

File: src/components/ProjectGridCard/ProjectGridCard.tsx [hunk 2]
```diff
@@ -53,16 +53,16 @@ export const ProjectGridCard = ({
             <Tag>
               <Text size="xsLowBold" color="gray.800">
                 {formatMessage(
-                  { id: 'sections.projectCardGrid.certified' },
-                  { year: project.certificationDate.getFullYear() }
+                  { id: 'components.projectGridCard.certified' },
+                  { year: new Date(project.certificationDate).getFullYear() }
```
Comment (lukasbals) [inline src/components/ProjectGridCard/ProjectGridCard.tsx:57]: We need to format the date string to a date obect before using `getFullYear`

**Comment by lialila** [comment]
can we also replace the `<FormattedMessage />` to `formatMessage` in `Timeline` component in this PR?

**Comment by lukasbals** [comment]
> can we also replace the `<FormattedMessage />` to `formatMessage` in `Timeline` component in this PR?

Would be awesome if you change that in a new PR? 🙂

**Review by lialila — COMMENTED** [review]
In `ProjectGridCard`  we need to replace  in `<Image` the `objectFit: project.thumbnail?.objectFit || 'cover' with `objectFit: 'cover'`

**Comment by lukasbals** [comment]
> In `ProjectGridCard` we need to replace in `<Image` the `objectFit: project.thumbnail?.objectFit || 'cover'` with `objectFit: 'cover'`

I don't think so. I think we should still leave it configurable in Strapi.

**Comment by lialila** [comment]
> > In `ProjectGridCard` we need to replace in `<Image` the `objectFit: project.thumbnail?.objectFit || 'cover'` with `objectFit: 'cover'`
> 
> I don't think so. I think we should still leave it configurable in Strapi.

Yes, but the `cover` version is the design version and `contain` version probably won't be used. Don't you think that it can be a bit confusing for the marketing team?

**Comment by lukasbals** [comment]
> > > In `ProjectGridCard` we need to replace in `<Image` the `objectFit: project.thumbnail?.objectFit || 'cover'` with `objectFit: 'cover'`
> > 
> > 
> > I don't think so. I think we should still leave it configurable in Strapi.
> 
> Yes, but the `cover` version is the design version and `contain` version probably won't be used. Don't you think that it can be a bit confusing for the marketing team?

I think it's confusing if you have the option to configure it in Strapi but in the end it's forced in the strapi-slices and nothing changes when you change it.

**Review by lialila — APPROVED** [review]

**Pull request merged**